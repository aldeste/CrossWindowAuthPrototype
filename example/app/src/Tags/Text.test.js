import Text from "./Text";
import renderer from "react-test-renderer";
import React from "react";
import "jest-styled-components";

describe("Text tag", () => {
  it("Should be defined", () => {
    expect(Text).toBeDefined();
  });

  it("Is a styled component", () => {
    expect(Text.name).toBe("StyledComponent");
  });

  it("Matches earlier configuration", () => {
    const tree = renderer.create(<Text />).toJSON();
    expect(tree).toMatchStyledComponentsSnapshot();
  });
});
