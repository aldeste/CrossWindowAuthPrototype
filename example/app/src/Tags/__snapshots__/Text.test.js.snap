// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Text tag Matches earlier configuration 1`] = `
.igPQVE {
  font-family: -apple-system, BlinkMacSystemFont, "Segoe UI", Roboto, Helvetica, Arial, sans-serif, "Apple Color Emoji", "Segoe UI Emoji", "Segoe UI Symbol";
}

<p
  className="sc-bdVaJa igPQVE"
/>
`;
